app/views/posts/index.html.erb

```diff
@@ -1,2 +1,38 @@
-<h1>Posts#index</h1>
-<p>Here are list of posts by user <%= params[:user_id]%> </p>
+<div class="user-details">
+  <img src=<%= @user.photo %> alt="<%= @user.name%>'s profile picture" />
+  <div class="user-info">
+    <p class="user_name"><%= @user.name %></p>
+    <p class="user_post-counter">Number of posts: <%= @user.posts_counter %></p>
+  </div>
+</div>
+<ul class="user-posts">
```

Comment:
- [ ] OPTIONAL: Post index page should show all posts written by user. Even though I created 5 posts written by user 1, on the posts index page I can see only 2 posts:
<img width="1339" alt="Screenshot 2022-05-03 at 23 08 46" src="https://user-images.githubusercontent.com/77038610/166525142-9d2bc6b0-97e7-49d1-9962-22d43da7fa6f.png">